plugins/metalink/utils.c

```diff
@@ -720,128 +720,146 @@ TDNFParseUrlTag(
     goto cleanup;
 }
 
-uint32_t
-TDNFXmlParseData(
-    TDNF_ML_CTX *ml_ctx,
-    xmlNode *node,
-    const char *filename
-    )
-{
-    uint32_t dwError = 0;
-    xmlChar* xmlContValue = NULL;
+void TDNFXmlParseStartElement(void *userData, const char *name, const char **attrs){
```

Comment:
follow the function definition style, like done in other places

Reply:
Done